docs/source/api_ref_training.rst

```diff
@@ -50,12 +50,9 @@ Utilities for enabling and working with distributed training.
     :toctree: generated/
     :nosignatures:
 
-    FSDPPolicyType
     init_distributed
     is_distributed
     get_world_size_and_rank
-    get_full_finetune_fsdp_wrap_policy
```

Comment:
There's a reference to this in the [QAT tutorial](https://github.com/pytorch/torchtune/blob/ac14e96443e60fd8ffed434637357ce22a9c5bff/docs/source/tutorials/qat_finetune.rst?plain=1#L173) too.